skins/base.css

```diff
@@ -1,13 +1,13 @@
 @media all {
 /* Main tags */
-body {font-family:arial,helvetica,sans-serif; font-size:75%; margin:0; -webkit-text-size-adjust:none; padding-top:0px!important; min-width:320px}
+body {font-family:Courier New,sans,serif,arial,helvetica,sans-serif,monospaced; font-size:75%; margin:0; -webkit-text-size-adjust:none; padding-top:0px!important; min-width:320px}
```

Comment:
This is in the base stylesheet, so I feel that forcing skins to work around a specific typeface might not be the greatest idea in the way of letting skins be customizable.